crates/slasher/src/lib.rs

```diff
@@ -216,7 +218,38 @@ where
             to_block
         );
 
-        for (log, _) in logs {
+        for (log, log_data) in logs {
+            // TODO(willpote): Remove, or make more resilient.
+            // Note this logic is not full proof. It will not handle lockRequestWithSignature
+            // nor if the lockRequest calls were for example, made via a proxy contract.
+            // This is a temporary solution to avoid slashing requests from the team's broker.
+            let tx_hash = log_data.transaction_hash.unwrap();
+            let tx = self
+                .boundless_market
+                .instance()
+                .provider()
+                .get_transaction_by_hash(tx_hash)
+                .await?
+                .unwrap();
```

Comment:
Feels like it might be more ideal to just skip the check instead of panicking if the provider does not provide the transaction hash with the log? Not sure if there are other conditions a provider might not include the tx hash with the log

Doesn't really matter since it's just a slashing bot, though